src/gps/nmea_parser.cc

```diff
@@ -0,0 +1,137 @@
+// BoB robotics includes
+#include "common/macros.h"
+#include "gps/nmea_parser.h"
+
+namespace BoBRobotics {
+namespace GPS {
+
+void
+parseTimeField(const std::string &field, std::tm &time)
+{
+    time.tm_hour = stoi(field.substr(0, 2));
+    time.tm_min = stoi(field.substr(2, 2));
+    time.tm_sec = stoi(field.substr(4, 2));
+
+    // Indicates that information about DST is not available
```

Comment:
Nope. It should be set to 0 if it isn't DST and 1 if it is. -1 just means we don't know (which is true because there's no way to signal this with NMEA messages).

I think in our case -1 will be functionally equivalent to 0, but the semantics are slightly different.